standalone/src/models/custom-application/custom-application-permission/custom-application-permission-draft/types.ts

```diff
@@ -1,10 +1,9 @@
 import type { TBuilder } from '@/core';
 import type { TMcSettingsCustomApplicationPermissionDataInput } from '@/graphql-types';
 
-export type TCustomApplicationPermissionDraft =
+export type TCustomApplicationPermissionDraftGraphql =
```

Comment:
It seems we're renaming an exported type which can lead to consumers failing, right?

Perhaps we need to also keep the previous one with a deprecation comment.